Classes/ViewHelpers/ImageViewHelper.php

```diff
@@ -6,7 +6,7 @@
 use TYPO3\CMS\Core\Resource\Exception\ResourceDoesNotExistException;
 use SMS\SmsResponsiveImages\Utility\ResponsiveImagesUtility;
 
-class ImageViewHelper extends \TYPO3\CMS\Fluid\ViewHelpers\ImageViewHelper
+class ImageViewHelper extends \TYPO3Fluid\Fluid\ViewHelpers\ImageViewHelper
```

Comment:
Are you sure that this is correct? I think that this is a TYPO3 ViewHelper.

Reply:
Seems you're right: https://api.typo3.org/master/class_t_y_p_o3_1_1_c_m_s_1_1_fluid_1_1_view_helpers_1_1_image_view_helper.html